### PR TITLE
Fix atmaero resources and turn test on for cheyenne.intel (replaces PR #1273)

### LINKED ISSUE #1241: Adjust resources for atmaero_control_p8; turn test on for cheyenne.intel
## Description
The default resources for the ``atmaero_control_p8`` test need to be increased for Cheyenne.intel. See PR #1218 for reference.

## Solution
Add a new default tasking setup for the ATMA app for use on all platforms. This can be the same as currently used (``TASKS_dflt``). For Cheyenne, override the defaults by specifying additional resources. The following works, but may not be optimal: 

layout=6,8
ATM_petlist_bounds:             0 293
CHM_petlist_bounds:             0 287

### PR DESCRIPTION
**Pull request by DeniseWorthen**
# PR Checklist

- [x] This PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [x] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below.

- [ ] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsibility to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

Creates a set of default resources for the ``atmaero_control_p8`` and turn test back on for cheyenne.intel

### Issue(s) addressed

- fixes #1241 

## Testing

The new default resources were tested on hera.intel. The test reproduced the existing baseline. 

**NOTE**: A new baseline for the ``atmaero_control_p8`` test will need to be created on cheyenne.intel. All other platforms where the test exists will reproduce the current baseline.

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss2.intel
- [x] wcoss_cray
- [ ] wcoss_dell_p3
- [ ] opnReqTest for newly added/changed feature
- [ ] CI

## Dependencies

None; script level changes only

### CODE REVIEW

**Comment by jkbk2004** [comment]
@DeniseWorthen #1265 merged in, So you may update and we can start testing.

**Comment by DeniseWorthen** [comment]
@jkbk2004 I thought I replied, but can't find it anywhere. I'll let you know when the cheyenne.intel baseline is ready. In the meantime, we can start on the RDHPCS platforms.

**Comment by jkbk2004** [comment]
> @jkbk2004 I thought I replied, but can't find it anywhere. I'll let you know when the cheyenne.intel baseline is ready. In the meantime, we can start on the RDHPCS platforms.

sure!

**Comment by DeniseWorthen** [comment]
@jkbk2004 Cheyenne.intel baseline has been updated with new test.

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: jet
Compiler: intel
Job: RT
[RT] Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/978368110/20220625024509/ufs-weather-model
Please make changes and add the following label back: jet-intel-RT

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: jet
Compiler: intel
Job: RT
[RT] Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/978368110/20220625164513/ufs-weather-model
[RT] Error: Test hafs_regional_atm 102 failed in run_test failed 
[RT] Error: Test hafs_regional_atm_thompson_gfdlsf 103 failed in run_test failed 
[RT] Error: Test hafs_regional_atm_ocn 104 failed in run_test failed 
[RT] Error: Test hafs_regional_atm_wav 105 failed in run_test failed 
[RT] Error: Test hafs_regional_atm_ocn_wav 106 failed in run_test failed 
[RT] Error: Test hafs_regional_docn 107 failed in run_test failed 
[RT] Error: Test hafs_regional_docn_oisst 108 failed in run_test failed 
[RT] Error: Test control_atmwav 123 failed in run_test failed 
[RT] Error: Test atmaero_control_p8 124 failed in run_test failed 
Please make changes and add the following label back: jet-intel-RT

**Comment by DeniseWorthen** [comment]
On wcoss-dell-p3, the hafs_regional_1nest_atm has hung right at the beginning three times.

```
20220626 150903.866 INFO             PET000 af FieldBundleRegridStore
20220626 150903.866 INFO             PET000 ... returned from wrtFB(05,01) FieldBundleRegridStore().
20220626 150903.866 INFO             PET000 bf FieldBundleRegridStore
20220626 150903.866 INFO             PET000 calling into wrtFB(06,01) FieldBundleRegridStore()....
```

and eventually in the log file:
```
TERM_RUNLIMIT: job killed after reaching LSF run time limit.
```

File: tests/default_vars.sh [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
   THRD=1
 
+  TASKS_atmaero=198 ; INPES_atmaero=4 ; JNPES_atmaero=8; WPG_atmaero=6
```
Comment (jkbk2004) [inline tests/default_vars.sh]: Sounds like parsing issue on jet: hafs_regional_docn and hafs_regional_docn_oisst. We need to remove the space before semi colons (TASKS_atmaero=198; INPES_atmaero=4;). With the fix, all tests run ok on jet.

Comment (DeniseWorthen) [reply tests/default_vars.sh]: @jkbk2004 Good catch!

**Review by jkbk2004 — APPROVED** [review]

**Comment by DeniseWorthen** [comment]
@jkbk2004 You made the fix in default_vars to run the test on jet, but did not push it?

**Comment by jkbk2004** [comment]
> @jkbk2004 You made the fix in default_vars to run the test on jet, but did not push it?

@DeniseWorthen I will push and then merge.

**Pull request merged**